### ISSUE TITLE
Incorrect relative URLs

### ISSUE THREAD

**Issue by chadly** [guybedford/require-css#126]
I've been using an older version of require-less / require-css and just recently updated to the latest version (v0.1.2). However, now all of my relative URLs are no longer correct.

In my less file, I have a background-image defined as:

``` css
background: url("../img/indicator.gif");
```

In debug mode, I was able to get my images to load correctly by specifying:

``` js
window.less = {
  relativeUrls: true
};
```

This correctly writes the image path to be relative to the the page that require-less is running from.

However, when I run the requireJS build and load the page with the css minified into the script file, the URLs are mostly correct, but missing a leading slash.

So instead of `/content/img/indicator.gif`, it writes it as `content/img/indicator.gif`. We use the script which loads the less file in question from many different subdirectories of our app, so the path defined doesn't work.

However, even if a leading slash were to be added to that path -- while it would work, it wouldn't be ideal. The actual path I want is `/content/v3.4/img/indicator.gif`. We define some virtual directories for cache busting so that our less file lives in this fake `/v3.4/` subdirectory. During the build, these directories don't exist and so require-less normalizes the URL based on the path on disk which does not include the `v3.4`.

The older version of require-less handles this scenario just fine since it didn't run the `normalize` part of the script until the page was loaded. The current version is running the `normalize` step as part of the build which is causing this problem.

tl;dr - I think there should be an option to allow normalize to not run until the page is loaded rather than being part of the build.

P.S. I opened this bug here instead of in require-less because it seems the change that broke this happened in `css-builder.js`. Let me know if I should open this issue at require-less instead.
